### DISCUSSION TITLE
Is parallel execution possible with Codeceptjs?

### DISCUSSION THREAD

**Discussion by s-sugumar** [codeceptjs/CodeceptJS#3854] [Q&A] [answered]
Anyone suggest me to run the parallel execution like we can do in selenium grid

**Comment by myrepojuly** [comment, marked as answer]
This is pretty straight forward. Start your selenium grid as usual. Set the `host: <the hub url>` and `port:<port number>` in the `codeceptjs` config. While running the tests use workers to set the number of workers needed based on the availability of your nodes.
